app/views/shared/code/_extract_method_5.html.erb

```diff
@@ -0,0 +1,17 @@
+def print_owing
+  outstanding = 0.0
+
+  # print banner
+  puts "*************************"
+  puts "***** Customer Owes *****"
+  puts "*************************"
+
+  # calculate outstanding
+  @orders.each do |order|
+    outstanding += order.amount
+  end
+
+  # print details
+  puts "name: #{@name}"
+  puts "amount: #{outstanding}"
+end
```

Comment:
New line at EOF. I'm confused, is this the view that gets shown to the user or is this a method that actually gets used within the app?